packages/gatsby-source-filesystem/src/create-remote-file-node-factory.js

```diff
@@ -0,0 +1,262 @@
+jest.mock(`fs-extra`, () => {
```

Comment:
Little confused here--this should be in __tests__ right? e.g.

`packages/gatsby-source-filesystem/src/__test__/create-remote-file-node-factory.js`